Chinese/大话西游.md

```diff
@@ -2,6 +2,12 @@
 
 前端攻城狮一枚。这枚攻城狮做出了很多网页，比如：
 
+  [google.com](https://google.com)
+
+  [github.com](https://github.com)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.